src/OpenConext/ProfileBundle/EventListener/LocaleListener.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright 2015 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenConext\ProfileBundle\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class LocaleListener implements EventSubscriberInterface
+{
+    /**
+     * @var string
+     */
+    private $defaultLocale;
+
+    public function __construct($defaultLocale)
+    {
+        $this->defaultLocale = $defaultLocale;
```

Comment:
It'd be good to be able to assert parameters adhere to our requirements, like we do in Conext Operations. (`Assert::string()`)